### ISSUE TITLE
A question about code

### ISSUE THREAD

**Issue by 963319431** [yehjin-shin/BSARec#10]
Hello author, I noticed that the forward function in the model folder fmlprec.py does not use extended-attention_mask=self. get.attenntion_mask (input_ids) to get the mask. May I ask why this is happening?
